src/services/expense.service.js

```diff
@@ -0,0 +1,80 @@
+const { Op } = require('sequelize');
+const { Expense } = require('../models/Expense.model.js');
+
+const getExpenses = async ({ userId, categoryList, from, to }) => {
+  const whereClause = {};
+
+  if (userId) {
+    whereClause.userId = userId;
+  }
+
+  if (categoryList && categoryList.length > 0) {
+    whereClause.category = { [Op.in]: categoryList };
+  }
+
+  if (from) {
+    whereClause.spentAt = { [Op.gte]: new Date(from) };
+  }
+
+  if (to) {
+    whereClause.spentAt = { ...whereClause.spentAt, [Op.lte]: new Date(to) };
+  }
+
+  const expenses = await Expense.findAll({
+    where: whereClause,
+  });
+
+  return expenses;
+};
+
+const createExpense = ({
+  userId,
+  spentAt,
+  title,
+  amount,
+  category = '',
+  note = '',
+}) => {
+  return Expense.create({
+    userId,
+    spentAt,
+    title,
+    amount,
+    category,
+    note,
+  });
+};
```

Comment:
The `createExpense` function should be marked as `async` to handle potential errors from the `Expense.create` call. This will allow you to use `try-catch` blocks to manage any database-related errors gracefully.